### DISCUSSION TITLE
Just found this fork....can I install along side main branch?

### DISCUSSION THREAD

**Discussion by PhilBaz** [smartavionics/Cura#140] [Q&A] [answered]
Hello,

I just found this fork. Can I install it along side the main branch 4.12?

Also, it says to save configuration. How do I go about doing that?

Thank you!!! can't wait to use hilbert :)

**Comment by 5axes** [comment]
Just a recomendation, never had any issue using Cura Master in parallel with the regular Cura releases

**Comment by smartavionics** [comment]
Hello @PhilBaz. Yes, as @5axes says, you can install my releases alongside the UM releases without problems.

As for backing up the configuration, it's a good idea to do that from time to time, especially when using a development release like mine because, sometimes, rarely, a change can break things and then it's nice to be able to get back to a working configuration.

Personally, being a technical type, I simply keep all of the configuration in a git repository. From time to time, especially if the UM devs have changed something that could cause problems, I make a snapshot of all of the configuration files. So it is easily possible for me to revert the configuration to any previously saved state. It's really very simple and has saved me some strife in the past. Alternatively, you could just zip up the folder containing the configuration files from time to time.

**Reply by PhilBaz** [reply]
Thank you Smart.  Okay, I'm on win 10 so if i just .zip and save the %APPDATA%\cura x.xx folders then Ill have my config backup.

Let me ask this, can I drop these config files into the equivalent folder of your fork and save myself the trouble of setting it up?

PS. very much looking forward to all the features in your fork, and thanks for all your efforts that i've already befitted from in the main branch!

**Comment by smartavionics** [comment, marked as answer]
Yes, you should be able to do that or simply copy all the files in the cura.x.y folder into the master folder that my release uses.

**Reply by PhilBaz** [reply]
Perfect! Thank you!
